platform/darwin/src/NSDictionary+MGLAdditions.mm

```diff
@@ -0,0 +1,51 @@
+#import "NSDictionary+MGLAdditions.h"
+
+#import "NSExpression+MGLAdditions.mm"
+
+@implementation NSDictionary (MGLAdditions)
+
+- (mbgl::PropertyMap)mbgl_propertyMap {
+    mbgl::PropertyMap propertyMap;
+    return [self propertyMapWithPropertyMap:propertyMap dictionary:self];
+}
+
+- (mbgl::PropertyMap)propertyMapWithPropertyMap:(mbgl::PropertyMap &)propertyMap dictionary:(NSDictionary *)dictionary {
```

Comment:
If the intent is to convert a dictionary into a property map, this could be a category method on NSDictionary that takes no parameters and returns a property map.